gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -583,4 +604,50 @@ public Completable insertOrUpdateOfflineArea(OfflineArea area) {
         .toSingle()
         .subscribeOn(schedulers.io());
   }
+
+  private Completable deleteTiles(OfflineAreaEntity offlineAreaEntity) {
+    return tileSourceDao
```

Comment:
Is there any way to split this stream up to make it more readable and self documenting?